emote/proxies.py

```diff
@@ -2,7 +2,6 @@
 Proxies are bridges between the world the agent acts in and the algorithm training loop.
 """
 
-from __future__ import annotations
 
 from typing import Dict, Protocol
```

Comment:
You don't want to update the type annotations? `Dict -> dict` etc.

Reply:
I was considering doing all files but it seems like something we can do incrementally.